go/vt/vtctl/reparentutil/emergency_reparenter.go

```diff
@@ -54,8 +54,11 @@ type EmergencyReparenter struct {
 // EmergencyReparentShard operations. Options are passed by value, so it is safe
 // for callers to mutate and reuse options structs for multiple calls.
 type EmergencyReparentOptions struct {
-	NewPrimaryAlias           *topodatapb.TabletAlias
-	IgnoreReplicas            sets.Set[string]
+	NewPrimaryAlias *topodatapb.TabletAlias
+	IgnoreReplicas  sets.Set[string]
+	// WaitAllTablets is used to specify whether ERS should wait for all the tablets to return and not proceed
+	// further after n-1 tablets have returned.
+	WaitAllTablets            bool
```

Comment:
I'm wondering if instead of a boolean this should be number of failures that can be tolerated. default to 1 but set to 0 for this particular case.
Between this and `IgnoreReplicas` and `numErrorsToWaitFor`, it is getting confusing to follow the logic.

Reply:
There are 2 reasons to keep this field as a boolean - 
1. We want the default to be such that the flow of ERS doesn't change. We can introduce this field as a boolean and start using it immediately because the default of `false` will not affect how ERS works. So adding this field in protobuf is going to be backward compatibility wise safe. Adding an integer would mean that we'll have to do something special for the default, because old vtctldclients will set the integer to 0, but we would still want to run ERS the default way i.e. with wait for only n-1 tablets.
2. Having the ability to set wait for tablets to 0 is required for correctness, but setting it to anything higher than 1 is only going to provide performance improvement, and not required for correctness. There is more chance of users shooting themselves in the foot with this capability and it is better to not add it right now, until we have a concrete reason for the same.